showcase/flexible/flexible-examples.js

```diff
@@ -44,12 +43,8 @@ const defaultProps = {
   margin: {top: 10, left: 10, right: 10, bottom: 10}
 };
 
-const FlexibleWidth = makeWidthFlexible(XYPlot);
-const FlexibleHeight = makeHeightFlexible(XYPlot);
-const FlexibleVis = makeVisFlexible(XYPlot);
-
-export function FlexibleCharts({height, width}) {
-  return (<div>
+export const FlexibleCharts = ({height, width}) => (
+  <div>
```

Comment:
this makes me a little worried, can you spin up the docs page and make sure this is rendering correctly in that enviroment?